storage/sql/migrate.go

```diff
@@ -274,4 +274,15 @@ var migrations = []migration{
 				add column code_challenge_method text not null default '';`,
 		},
 	},
+	{
+		stmts: []string{
+			`
+			create table if not exists blocked_user (
```

Comment:
how about invalid_login_attempts as table name instead of blocked_user?

Reply:
seems good, will update it.